st2tests/tests/selenium/selenium_sanity.py

```diff
@@ -0,0 +1,96 @@
+import datetime
```

Comment:
Do you mean for Selenium?
It's Apache 2.0 (http://www.seleniumhq.org/about/license.jsp)